docs/index.rst

```diff
@@ -118,5 +118,11 @@ processing, and data analysis.
 
             To the project details
 
+|User Stats|
+
 .. _feedback@astropy.org: mailto:feedback@astropy.org
 .. _affiliated packages: https://www.astropy.org/affiliated/
+
+.. |User Stats| image:: https://github.com/astropy/repo_stats/blob/cache/cache/astropy_user_stats_light.png?raw=true
```

Comment:
Our doc supports both light/dark mode. The light version looks out of place in dark mode. Is there a way to make this image auto-adjust to light/dark or is this the same problem as https://github.com/matplotlib/matplotlib/issues/25983 ?

Reply:
I had to make the image background of the transparent; updated version being used as of 
b51fb17
<img width="936" alt="Screenshot 2024-08-16 at 11 52 15 AM" src="https://github.com/user-attachments/assets/fb0d148c-0b62-4b8f-8478-bba364510870">

Reply:
The README looks fine in both light and dark, but my local build of the docs shows a white background around the image in dark mode; I don't know why, I'm guessing sphinx doesn't render transparency and instead sets it to white (as the RGB values of transparency are 255, 255, 255, the same as white; the A in RGBA provides transparency information). I don't have a good workaround for that.

Reply:
I am a little concerned that the contrast is very low on the readme in light mode. I don't think it would pass the accessibility contrast checker like it is at the moment. It certainly looks better in dark than light. (This is GH not docs fwiw)

Reply:
Thank you for noting this. [8db4075](https://github.com/astropy/astropy/pull/16822/commits/8db40755f0fcf468a2d580aaa1865e3b06f1ed83) resolves this by updating the image to match light or dark mode.